wiki/Beatmap_Editor/Menu/en.md

```diff
@@ -1,441 +1,256 @@
+| Flip Horizontally (Ctrl + H) | Flip the note(s) from left to right (with respect to y-axis line of Playfield origin) and vice versa. |
+| Flip Vertically (Ctrl + J) | Flip the note(s) from up to down (with respect to x-axis line of Playfield origin) and vice versa. |
+| Rotate 90° Clockwise (Ctrl + >) | Rotate the note(s) 90° right |
+| Rotate 90° Anticlockwise (Ctrl + <) | Rotate the note(s) 90° left |
+| Rotate by... (Ctrl + Shift + R) | Set your rotation angle (+/- 180°) for the poor note(s). You can adjust the rotation Direction [(Anti-/)Clockwise] and Origin [(Playfield/Selection) Centre]. |
+| Scale by... (Ctrl + Shift + S) | Set your enlargement size (0.500x - 1.200x) for the poor note(s). You can adjust the Origin [(Playfield/Selection) Centre] and "Apply to" (the enlargement scale to) either or both x-axis and/or y-axis. |
+
+### Reset Buttons
+
+| Name | Description |
+| ---- | ----------- |
+| Reset selected objects' samples | Remove hitsound of currently selected note. |
+| Reset all samples | Remove all hitsounds from all the notes in this difficulty. |
+| Reset all combo colour | Cancel all combo colourings on the notes. |
+| Reset breaks | Cancel breaktimes. Does not work on automated breaktimes that prevent excessive HP drain. |
+
+### Nudge commands
+
+| Name | Description |
+| ---- | ----------- |
+| Nudge Backward (J) | Move the note(s) one beat backward based on Beat Snap Divisor. |
+| Nudge Forward (K) | Move the note(s) one beat forward based on Beat Snap Divisor. |
 
 View
-====
-
-<img src="M_View.jpg" title="fig:Right|View menu" alt="Right|View menu" width="200" /> Places of interest.
-
-<table>
-<thead>
-<tr class="header">
-<th><p>View menu</p></th>
-</tr>
-</thead>
-<tbody>
-<tr class="odd">
-<td><p>colspan=2| Tabbing commands</p></td>
-</tr>
-<tr class="even">
-<td><p>Compose<br />
-(F1)</p></td>
-</tr>
-<tr class="odd">
-<td><p>Design<br />
-(F2)</p></td>
-</tr>
-<tr class="even">
-<td><p>Timing<br />
-(F3)</p></td>
-</tr>
-<tr class="odd">
-<td><p>colspan=2| Setup commands</p></td>
-</tr>
-<tr class="even">
-<td><p>Song Setup...<br />
-(F4)</p></td>
-</tr>
-<tr class="odd">
-<td><p>Timing Setup...<br />
-(F6)</p></td>
-</tr>
-<tr class="even">
-<td><p>colspan=2| Special commands</p></td>
-</tr>
-<tr class="odd">
-<td><p>Volume</p></td>
-</tr>
-<tr class="even">
-<td><p>Grid Level</p></td>
-</tr>
-<tr class="odd">
-<td><p>Show Video</p></td>
-</tr>
-<tr class="even">
-<td><p>Show Sample Name</p></td>
-</tr>
-<tr class="odd">
-<td><p>Snaking sliders</p></td>
-</tr>
-<tr class="even">
-<td><p>Hit animations</p></td>
-</tr>
-<tr class="odd">
-<td><p>Follow points</p></td>
-</tr>
-<tr class="even">
-<td><p>Stacking</p></td>
-</tr>
-<tr class="odd">
-<td><p>Disable Undo States</p></td>
-</tr>
-</tbody>
-</table>
+--------
+
+![View menu](M_View.jpg "View menu")
+
+Places of interest.
+
+### Tabbing commands
+
+| Name | Description |
+| ---- | ----------- |
+| Compose (F1) | Direct to Compose tab. |
```

Comment:
change `F1` to `` `F1` ``
do this for this for the rows below and the next table below